### DISCUSSION TITLE
爱情的纹理第00期 - 模板留存demo

### DISCUSSION THREAD

**Discussion by giscus** [WangEn/weekly#1] [General]
# 爱情的纹理第00期 - 模板留存demo

封面图来源于上周买的一个麻将，摸着温润如玉的感觉很是喜欢，今年春节打算把家里接到杭州过年，麻将会是一个特别适合过年的活动。

https://www.715721.xyz/posts/00-%E6%A8%A1%E6%9D%BF%E7%95%99%E5%AD%98demo

**Comment by WangEn** [comment]
Astro主题部署

> base from Tw93
